File: API/Models/ApplicationUser.cs
using Microsoft.AspNetCore.Identity;


namespace API.Models
{
    public class ApplicationUser : IdentityUser<Guid>
    {
        public string Name { get; set; }
        public override string UserName { get; set; }
        public override string NormalizedUserName { get; set; }
        public bool IsEnabled { get; set; }
        public bool IsDelete { get; set; }
        public string Avatar { get; set; }
    }

}

File: CMS/src/app/_core/_models/application-user.ts
export interface ApplicationUser {
    name: string;
    userName: string;
    normalizedUserName: string;
    userType: string;
    isEnabled: boolean;
    isDelete: boolean;
    adAccountMapping: string;
    id: string;
    email: string;
    normalizedEmail: string;
    emailConfirmed: boolean;
    passwordHash: string;
    securityStamp: string;
    concurrencyStamp: string;
    phoneNumber: string;
    phoneNumberConfirmed: boolean;
    twoFactorEnabled: boolean;
    lockoutEnd: string;
    lockoutEnabled: boolean;
    accessFailedCount: number;
}